### PR TITLE
Migrate staging domain name

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -208,13 +208,13 @@ def _check_repo_age():
     if time.time() - os.path.getmtime(REPO_OUTDATED_FILE) > REPO_OUTDATED_TIME:
         warn('Your fabric-scripts may be out-of-date. Please `git pull` the repo')
 
-def _set_gateway(name, hostname='alphagov.co.uk'):
```
Comment (mattbostock) [inline fabfile.py]: `jumpbox_domain` would probably be more descriptive.

Comment (alexmuller) [inline fabfile.py]: Updated!

**Comment by bradleywright** [comment]
Can we get this merged? This was confusing to me this morning on 2nd line.

**Comment by mattbostock** [comment]
:+1:

**Pull request merged**